### ISSUE TITLE
Search UI/Some screen for Web

### ISSUE THREAD

**Issue by passionate2023** [sqlitebrowser/dbhub.io#206]
Hi,

I'm looking for a SQLlite full text search with UI/Some Screen

Java 1.4 version will have 100 books in .txt files
Java 1.8 version will have 100 books in .txt files
Sherlock Homes stories where each chapter in .txt files
Addres from a-z each alphabet will have each .txt files
Website address a-z each alphabet will have each .txt files...
Java will be category and sub-category will be 1.4 or 1.8 etc.,
Story will be category and Each book will be a category

Is it possible to achieve in dbhub.io with any specific db or table structure to achieve the above

Thanks

**Comment by justinclift** [comment]
@passionate2023 Apologies for the delay, we somehow missed several issues that have been filed in this repository.

Did you figure out a solution for this? :smile:

**Comment by passionate2023** [comment]
No. @justinclift Please help me with a good one which has PUBLIC sharing URL with search option and highlight

**Comment by justinclift** [comment]
@passionate2023 Ahhh.  Do you have programming skills to create stuff yourself?
